atest/robot/tags/include_and_exclude.robot

```diff
@@ -73,6 +73,9 @@ Include with NOT
 Exclude with NOT
     --exclude excl1NOTexcl2ANDexcl3    Excl-123    @{INCL_ALL}
 
+Exclude with robot:exclude tag
+    --tag no-use    Robot-exclude
+    
```

Comment:
I assume this is the test that's failing? Tests in this suite validate what tests were actually run. The new test with `robot:exclude` should never be executed so testing that it's run isn't possible. That it's not run is actually tested already by other tests in this suite. For example, the first fest validates that all tests are run when `--include/--exclude` aren't used and the new test isn't run even in that case.

It would thus be enough to just add the new test with `robot:exclude` to the `atest/testdata` side, but it would nevertheless be nice to somehow make it clear on the `atest/robot` side that this functionality is tested. Possibly it would be enough to just add a documentation to the first test that the test with `robot:exclude` is always excluded automatically.